pkg/fission-cli/cmd/spec/apply.go

```diff
@@ -185,6 +188,27 @@ func (opts *ApplySubCommand) run(input cli.Input) error {
 	return nil
 }
 
+func warnIfDirtyWorkTree(path string) {
+	repo, err := git.PlainOpen(path)
```

Comment:
Would this work if a user is not using Git in his project or it's just a local folder and not git repo. Can we check if they have enabled Git or not in their project.

Reply:
It would return an error which I ignore and return from that point, as you pointed above

Reply:
As per documentation 
https://pkg.go.dev/gopkg.in/src-d/go-git.v4#PlainOpen
PlainOpen calls returns ErrRepositoryNotExists if not valid repository, we should ignore this error only. Rest we should log.

Reply:
Okay